protobuf/transaction.proto

```diff
@@ -102,7 +80,9 @@ message Transaction {
             Type type = 2;
             Options options = 3;
         }
-        message Res {}
+        message Res {
+            int32 processingTimeMillis = 1;
```

Comment:
When a client is measuring network latency, they should subtract this value in their calculation. This tells the client how long the server spent doing the work to open the Transaction.